issue2pr.py

```diff
@@ -9,10 +9,27 @@
 
 from __future__ import print_function
 
+import argparse
+import getpass
+import json
```

Comment:
Looking into it a bit more, I *think* you can actually do ``json=`` instead of ``params=`` and it will auto-encode the data to JSON (in requests >= 2.4.2)